src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -248,12 +307,12 @@ var ReactTestFiberRenderer = {
       createNodeMock,
       tag: 'CONTAINER',
     };
-    var root = TestRenderer.createContainer(container);
-    TestRenderer.updateContainer(element, root, null, null);
+    var root = TestRenderer.createContainer(container).stateNode;
```

Comment:
After https://github.com/facebook/react/pull/8934 this doesn't need stateNode anymore.